crates/cxx-qt-gen/src/parser/mappings.rs

```diff
@@ -1,225 +0,0 @@
-// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
-// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
-//
-// SPDX-License-Identifier: MIT OR Apache-2.0
-
-use std::collections::BTreeMap;
-
-use quote::format_ident;
-use syn::{spanned::Spanned, Attribute, Ident, Path, Result};
-
-use crate::syntax::{attribute::attribute_find_path, expr::expr_to_string, path::path_from_idents};
-
-#[derive(Default, Debug)]
-pub struct ParsedCxxMappings {
```

Comment:
renaming / moving this struct could have been done in a separate commit to ease review :-)

Reply:
Sorry about that :see_no_evil: